bitnami/kafka/values.yaml

```diff
@@ -11,6 +11,8 @@
 ## @param global.imagePullSecrets Global Docker registry secret names as an array
 ## @param global.defaultStorageClass Global default StorageClass for Persistent Volume(s)
 ## @param global.storageClass DEPRECATED: use global.defaultStorageClass instead
+## @param global.ipFamily Global IpFamilies
+## @param global.ipFamilyPolicy Global IpFamilyPolicy Services
```

Comment:
Please don't use the global key to set this new value. We use this key for cross chart configuration and at the moment this feature (`ipFamilies`) is only supported by a few charts.

Could you follow the same approach followed in [nginx-ingress-controller](https://github.com/bitnami/charts/blob/3d4d32a8c1b8fcffd0268229bc0ee5197788fc3a/bitnami/nginx-ingress-controller/values.yaml#L926-L935) (setting the value in the `service` key)